omod/src/main/java/org/openmrs/module/commonlabtest/web/controller/LabTestOrderPortletController.java

```diff
@@ -67,10 +68,13 @@ protected void populateModel(HttpServletRequest request, Map<String, Object> mod
 						orderJsonArray.add(childJsonObject);
 					}
 				}
+				boolean anyTestRequireSample = false;
+				for (JsonElement element : orderJsonArray) {
+					anyTestRequireSample = ((JsonObject) element).get("requiredSpecimen").getAsBoolean();

```

Comment:
Not making sense. If there are 2 elements; the first one has requiredSpecimen = true and second one has requiredSpecimen = false then anyTestRequireSample will be false when the loop ends. Are we missing a break statement, or some other operation is to be done inside the loop?

Reply:
Fixed! probably an undo resulted in this.